### ISSUE TITLE
Relationship between LayerIntegratedGradients & configure_interpretable_embedding_layer

### ISSUE THREAD

**Issue by LopezGG** [pytorch/captum#460]
Hi,
I am working with the tutorial [here](https://captum.ai/tutorials/Bert_SQUAD_Interpret). I took the example 

> question, text = "What is important to us?", "It is important to us to include, empower and support humans of all kinds."

and tried to see the effect of the word "What" on the start position using
`attributions_start_sum[1]`
which is 
> tensor(0.3861, device='cuda:0', dtype=torch.float64, grad_fn=<SelectBackward>)
Then I ran the code to get the effect of word, token_type and position embeddings and the attributions look like
![image](https://user-images.githubusercontent.com/1724189/91678550-b3f31f80-eafa-11ea-9719-c363f1e0eac5.png)

for "What" 
>0.02 + (-0.01) + 0.07 != 0.3861

Is this expected ? Could you please help me understand this

**Comment by NarineK** [comment]
Hi @LopezGG, Is the question that you were expecting `0.02 + (-0.01) + 0.07` to be equal to `0.3861`?

I'm not sure that we can make that type of assumption. Due to the interaction effects of the features that might not always hold.
The numbers you see might be a bit different because you might be using a different version of Bert model that was trained with different hyper parameters.

**Comment by LopezGG** [comment]
Thank you @NarineK . Yeah I was hoping to understand the different components that add up to give 0 `0.3861` . I will spend sometime this week to figure out the the different interactions. Any pointers to resources will be very helpful

**Comment by NarineK** [comment]
@LopezGG , I'd also look into LayerNorm and Dropout that is happening after concatenating those layers. 
https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py#L213

If you attribute to the inputs of the `LayerNorm` you might get the result that you're expected ? Let me know if you get chance to try that.

**Comment by LopezGG** [comment]
@NarineK : Thank you !  I am looking into the code and trying to understand this. I will get back to you either with more questions or some answers :)

**Comment by NarineK** [comment]
@LopezGG, is this still an outstanding issue for you ?

**Comment by LopezGG** [comment]
Not looking into this right now. thank you Narine
